### ISSUE TITLE
Test Case: Verify Feature Card Description

### ISSUE THREAD

**Issue by popkhadzeg** [NCIOCPL/cgov-digital-platform-qa#39]
# Pre-requisites
* YAML content is ready for all the content types mentioned below

# Content Types Covered:
* Article
* Blog Posts
* Press Release
* Cancer Center
* Biography
* Event
* Mini      Landing Page
* Home &      Landing Page
* Cancer Type      Home Pages
* Blog Landing Page     (Series page)
* Infographic
* Video
* File

# Steps:
1. Verify that the Feature card description is visible if Feature card description field is populated on the card on the content.
2. Verify that the Feature card description is not visible if Feature card description field is empty on the card on the content.
3. Verify that the correct text is displayed on the Feature Card description.

Depends on Issue #<number>

###Requirement Reference
https://github.com/NCIOCPL/cgov-digital-platform/wiki/Common-Fields#Feature-Card-Description

**Comment by kate-mashkina** [comment]
PR#85 resolving this issue has been merged into master
